DemoqaProject/config/Config.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+namespace DemoqaProject
+{
+    public class Config
+    {
+        public static string baseURL = "https://demoqa.com/";
+     
+            public static string elementsText = "Elements";
```

Comment:
These lines shouldn't be in config I believe. Config is more for driver configuration. Please create a separate test fixture enum/class to store this static data.

Reply:
done